RoboJackets/robocup-software
# Zoner defense

## Description
Implement zone defense.

## Steps to Test
1. Force robot factory position to be zoner.
2. Try moving ball around and observe zoner movement

**Expected result:**
It should stay in assigned point when no enemies in range, mark if one enemy robot, and stay in centroid if multiple enemies in range.

## Key Files to Review
 * zoner.cpp
 * zoner.hpp

## Review Checklist


## (Optional) Sub-issues (for drafts)
_Note: if you find yourself breaking this PR into many smaller features, it may make sense to break up the PR into logical units based on these features._
